src/pages/accounts/resetPassword/styles.scss

```diff
@@ -0,0 +1,126 @@
+@import '../../../../styles/variables.scss';
+$container-background: $white-two;
+
+.account__container {
+  height: 100vh;
+  width: 100vw;
+  color: var(--black);
```

Comment:
use variables in sass, this is normal CSS, and yet `--black` is not defined here